src/jitlayers.cpp

```diff
@@ -604,7 +604,7 @@ CodeGenOpt::Level CodeGenOptLevelFor(int optlevel)
 #ifdef DISABLE_OPT
     return CodeGenOpt::None;
 #else
-    return optlevel < 2 ? CodeGenOpt::None :
+    return optlevel < 1 ? CodeGenOpt::None :
```

Comment:
`optlevel == 1` will now give `CodeGenOpt::Aggressive`. Is that intended?

Reply:
Oh, yeah that's wrong. I want Default. The nested conditionals messed me up